capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.Capitalize
```

Comment:
- [ ] I think this will give you an error. Kindly use `capitalize` instead of `Capitalize`.